specifyweb/frontend/js_src/lib/components/DataModel/__tests__/collection.test.ts

```diff
@@ -52,7 +52,7 @@ describe('fetchCollection', () => {
     resource_uri: getResourceApiUrl('Locality', 1),
   };
   overrideAjax(
-    '/api/specify/locality/?limit=1&localityname=Test&orderby=-latlongaccuracy&yesno1=True',
+    '/api/specify/locality/?limit=1&localityname=Test&orderby=-latlongaccuracy&yesno1=True&domainfilter=false',
```

Comment:
this change should be reverted as `domainfilter=false` should be equivalent to not having that value
if it is not, then it's a back-end bug, that should be fixed on the back-end, not front-end

if nothing else, think of this - this change makes every single resource request in specify a few bites larger. and if there is a bug there where `&domainfilter=false` is not the same as omitting it, it should be fixed at the back-end to benefit people who are using the API directly (and we do have such)